package.json

```diff
@@ -51,7 +51,7 @@
     "wd": "^0.4.0"
   },
   "devDependencies": {
-    "jasmine-node": "~1",
-    "jshint": "^2.6.0"
+    "jshint": "^2.6.0",
+    "appium": "1.5.3"
```

Comment:
Adding Appium to dependencies would drastically increase paramedic install time for every build on CI, while only a small subset of them actually needs it.
I think we can still install this particular version via executing `npm i appium@1.5.3` in `installAppiumServer()` function, aren't we?